lib/tasks/events.rake

```diff
@@ -0,0 +1,45 @@
+namespace :events do
+  desc 'Nullifies wrong foreign keys for Track, Room, DifficultyLevel'
+
+  task fix_wrong_track: :environment do
```

Comment:
My logic was 3 different things, 3 different tasks. I have put everything in one task.